generated/nifake_non_ivi/nifake_non_ivi_service.cpp

```diff
@@ -673,5 +676,16 @@ namespace nifake_non_ivi_grpc {
     }
   }
 
+  bool NiFakeNonIviService::is_enabled()
+  {
+    return feature_toggles_.is_enabled;
+  }
+
+  NiFakeNonIviService::NiFakeNonIviFeatureToggles::NiFakeNonIviFeatureToggles(
+    const nidevice_grpc::FeatureToggles& feature_toggles)
+    : is_enabled(
+        feature_toggles.is_feature_enabled("nifake_non_ivi", CodeReadiness::kNextRelease))
```

Comment:
Just checking, this one is marked `NextRelease` since it's testing DAQ specific non-ivi types, right?

Reply:
Yeah, I marked both daq and fake non ivi

```python
    'code_readiness': 'NextRelease',
```

Just so we can use fake non ivi to test that state.